Makefile

```diff
@@ -22,12 +22,38 @@ pip-compile: ## update the requirements/*.txt files with the latest packages sat
 	pip-compile --upgrade requirements/base.in
 	pip-compile --upgrade requirements/dev.in
 	pip-compile --upgrade requirements/doc.in
-	pip-compile --upgrade -o requirements/test.txt requirements/base.in requirements/doc.in requirements/test.in
+	pip-compile --upgrade requirements/test.in
 ifneq (, $(wildcard $(PRIVATE_IN)))
 	pip-compile --upgrade $(PRIVATE_IN)
 endif
 
 
+rq-cmd:
+	$(eval remote ?= heroku)
+	$(cmd) -u $(shell heroku config:get REDIS_URL -r $(remote))
+
+
+rq-dashboard:
+	@$(MAKE) rq-dashboard-open &
+	@$(MAKE) cmd="rq-dashboard" rq-cmd
+
+
+rq-dashboard-open:
+	$(eval url = http://localhost:9181)
+	@until $$(curl -o /dev/null --silent --head --fail $(url)); do\
+		sleep 1;\
+	done
+	open http://localhost:9181
+
+
+rq-requeue-failed:
+	@$(MAKE) cmd="rq requeue -a" rq-cmd
+
+
+rqinfo:
+	@$(MAKE) cmd=rqinfo rq-cmd
```

Comment:
I don't understand all the make-magic here.  There really wasn't a simpler way to achieve what we need?